### PR TITLE
Setup logging

### PR DESCRIPTION
**Pull request by Arthelon**
Addresses #27

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,19 +18,26 @@ ci:
...
 	@docker push ${APP_IMAGE_SHA}
 
 run-migrations:
 	@kubectl -n gitlint-bot delete job/gitlint-bot-db-migrations --ignore-not-found
 	@kubectl -n gitlint-bot apply -f ./deploy/migrations-job.yaml
 	@kubectl -n gitlint-bot wait --for=condition=complete job/gitlint-bot-db-migrations --timeout=30s
+
+upload-sourcemaps:
+	@npm run build
+	@sentry-cli releases new ${SENTRY_RELEASE} --project ${SENTRY_PROJECT}
```
Comment (kiootic) [inline Makefile:40]: Let's just use plain DSN without source mapping/release for now.

**Comment by Arthelon** [comment]
Seems to work without providing a DSN.

File: src/listeners.ts [hunk 2]
```diff
@@ -3,17 +3,18 @@ import Webhooks from "@octokit/webhooks";
...
 async function processCommit(
   githubCommit: GithubCommit,
   repoName: string
 ): Promise<CommitInfo> {
-  Logger.captureMessage("Receive new commit", (scope) =>
-    scope.setExtra("data", {
-      repoName,
-      hash: githubCommit.id,
-    })
+  console.log(
```
Comment (kiootic) [inline src/listeners.ts:12]: For logging, we should be using structured logging. You can refer to https://probot.github.io/docs/logging/ for details.

File: src/listeners.ts [hunk 3]
```diff
@@ -26,9 +27,16 @@ async function processCommit(
 export async function onPush(
   context: Context<Webhooks.WebhookPayloadPush>
 ): Promise<void> {
+  addInvocationBreadcrumb("'onPush' Github Webhook push event handler");
   const { commits, repository } = context.payload;
   const commitInfos = await Promise.all(
-    commits.map(async (commit) => processCommit(commit, repository.full_name))
+    commits.map(async (commit: GithubCommit) => {
+      context.log.info("Received new #commit: %O", {
```
Comment (kiootic) [inline src/listeners.ts:34]: This way the object is included within the message. That object should be passed as the first parameter, so the log entry JSON would look like this:
```json
{
    "msg": "Received new commit",
    "repoName": "...",
    "hash": "..."
}
```

File: src/listeners.ts [hunk 4]
```diff
@@ -26,9 +27,17 @@ async function processCommit(
 export async function onPush(
   context: Context<Webhooks.WebhookPayloadPush>
 ): Promise<void> {
+  addInvocationBreadcrumb("'onPush' Github Webhook push event handler");
   const { commits, repository } = context.payload;
   const commitInfos = await Promise.all(
-    commits.map(async (commit) => processCommit(commit, repository.full_name))
+    commits.map(async (commit: GithubCommit) => {
+      context.log.info({
```
Comment (kiootic) [inline src/listeners.ts:34]: A most idomatic way is to write:
```js
context.log.info({repoName: '...', hash: '...'}, "received a new commit")
```

File: Makefile [hunk 5]
```diff
@@ -34,3 +34,10 @@ run-migrations:
...
```
Comment (kiootic) [inline Makefile:40]: In case you miss this: please remove the Sentry release flow, we don't need this at the moment:

**Pull request merged**